YaLibs/YaToolsIDALib/Repository.cpp

```diff
@@ -750,20 +769,34 @@ bool Repository::rebase(const std::string& upstream, const std::string& destinat
 
 bool Repository::add_file_to_index(const std::string& path)
 {
+    fs::path p = path;
+    if (!include_idb_ &&
+        (path == get_current_idb_name() ||
+        path == get_original_idb_name())) {
+        p.replace_extension(".idx_file");
```

Comment:
I would just add an empty .gitignore file and remove the weird .idx_file

Reply:
fixed in 1d1e98c73c4b5c6eec04e07bac56ccb00b47b515

Reply:
That makes way more sense =D

Should it be empty though, or should it contain the IDB name?

Reply:
Empty or the IDB(_local) name looks good imho

Reply:
Went with IDB(_local) in af2b97e0a215bedd92fe3bb036af8012f5654736